File: CursoCsharpUdemy/exer/outros/Exer07.cs
/*
*   6) Ler uma temperatura em graus Fahrenheit e apresentá-Ia convertida em graus Celsius. A fórmula de conversão de
*   temperatura a ser utilizada é C = (F - 32) * 5 / 9, em que a variável F representa é a temperatura em graus
*   Fahrenheit e a variável C representa é a temperatura em graus Celsius.
*/

using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace exer.outros
{
    public class Exer07
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Qual a temperatura ? (em fahrenheit)");
            double fahrenheit = double.Parse(Console.ReadLine());

            double celsius = (fahrenheit -32) * 5 / 9;

            Console.WriteLine("Essa temperara em Fahrenheit será: " + celsius);
        }
    }
}

File: CursoPython3/exer/lista64Exer/exer07.py
"""
7) Ler uma temperatura em graus Fahrenheit e apresentá-la convertida em graus Celsius. A fórmula de conversão de 
temperatura a ser utilizada é C = (F - 32) * 5 / 9, em que a variável F é a temperatura em graus Fahrenheit e a 
variável C é a temperatura em graus Celsius. 
"""

fahrenheit = float(input("Informe a temperatura em Fahrenheit: "))

celsius = (fahrenheit - 32) * 5 / 9

print("Temperatura em Celsius: ", celsius)
